capstone/src/main/webapp/dashboard.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Dashboard</title>
+    <!-- TODO(biancamacias): revist url to add api key -->
+    <!-- TODO(biancamacias): reference style.css and script.js -->
+  </head>
+  <body>
+    <div id="header">
+      <h1>UniViz</h1>
+    </div>
+    <div id="map">
+    </div>
+    <div id="colleges">
+    </div>
+    <div id="data">
```

Comment:
No action for this PR. But i think `data` is not very specific. I would pick more specific names for element IDs. Like `tuition-data` or something like that.